main.py

```diff
@@ -1,116 +1,99 @@
-# Import Pyton Libraries ---------------------------------------------------
```

Comment:
Import "Pyton" libraries. oh is that what `import` means?

Let's get rid of that.